app/command.go

```diff
@@ -155,10 +155,17 @@ func (a *App) ListAllCommands(teamId string, T goi18n.TranslateFunc) ([]*model.C
 }
 
 func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	parts := strings.Split(args.Command, " ")
-	trigger := parts[0][1:]
+	message := ""
+	trigger := ""
+	if strings.ContainsAny(args.Command, " \n\t\v\r\f") {
```

Comment:
It should suffice to only invoke `IndexAny` which already returns `-1` if no matches are found.

Reply:
Should we consider finding a way to rely on `IsSpace` as used inside [`strings.TrimString`](https://golang.org/src/strings/strings.go?s=21924:21955#L861) to match instead?

Reply:
suggestions taken!